package.json

```
@@ -18,7 +18,8 @@
     }
   },
   "peerDependencies": {
-    "@sinclair/typebox": ">=0.26 <=0.33"
+    "@sinclair/typebox": ">=0.26 <=0.33",
+    "fastify": "^5.0.0-alpha.3"
```

I suppose this can be changed to 5.x now?
```suggestion
    "fastify": "5.x"
```

---

Thank you, I changed it to standard semver notation.